ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/defaults/main.yml

```diff
@@ -36,3 +36,6 @@ ocp4_workload_gitea_operator_ssl_route: true
 
 # Route name for the Gitea Route. Only set if you know what you are doing!
 # ocp4_workload_gitea_operator_gitea_route: gitea.apps.clusterdomain
+
+ocp4_workload_gitea_operator_admin_user: gitea
```

Comment:
Should we have a variable to enable/disable creation of admin users? E.g. ocp4_workload_gitea_operator_install_admin_user: true

Reply:
I would think that you would always want an admin user. Without an admin user you can't create actual users, people would have to register, and etc.

I can't envision a situation where having the admin user would be something you didn't want.